### DISCUSSION TITLE
How does one specify in which schema the _prisma_migrations table goes?

### DISCUSSION THREAD

**Discussion by GunterMueller** [prisma/prisma#21452] [Q&A]
### Question

Hello,
How does one specify in which schema the _prisma_migrations table goes?
Thanks.

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by janpio** [comment]
You don't, it should be either the only schema Prisma is talking to, or the first one when you are using the `multiSchema` preview feature.

**Reply by hunner** [reply]
> You don't, it should be either the only schema Prisma is talking to, or the first one when you are using the `multiSchema` preview feature.

On postgresql, it appears that with `multiSchema` enabled and only a single DB schema specified in `datasource.schemas`, Prisma is creating the `_prisma_migrations` table where the `datasource.url` schema query parameter (e.g. `?schema=oops`) points, or in the default schema (`public`) if none in specified in the `datasource.url` connection string.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
